### ISSUE TITLE
Parity not working properly after few days of running and needs restart

### ISSUE THREAD

**Issue by bgadmin** [paritytech/parity#5611]
After some random time period, a couple of days maybe, parity misbehaves. I've noticed this problem when I try to create a new account through my application. 

When the new account is created in application it automatically creates ethereum account and sends some ether to it.

Accounts get created but ether is never sent to the new account. I didn't get to catch the log for send ether transaction.

Basically, when this happens I don't know what to do but good old restart of parity resolves the issue. Of course tried transactions are lost.

**Comment by 5chdn** [comment]
Could you explain _expected behavior_ versus _actual behavior_? How does parity misbehave?

Also, which version are you running, and which network is configured? How does your configuration/cli look like?

Finally, could you explain why a new account-creation process should send some ether to it? Do you have some script for this or are you referring to mining?

**Comment by bgadmin** [comment]
@5chdn OK so let me explain scenario first:

- No, we are not mining, we have email service written in nodejs that interacts with parity and ipfs. 

- In register process, new ethereum account is created for the new user and he gets some ether so that he can send emails. 

- The expected behaviour is when both of these "ethereum" actions are completed:
     1. Register ethereum account/addr
     2. Send Ether to the new account

- Unexpected behaviour is when user registers (1. Register ethereum account/addr) and ether is never sent. 

- In application logs everything looks as expected, the account gets created, application tries to send ether but transaction never enters ethereum. 

- Version: Parity/v1.6.6-beta-8c6e3f3-20170411/x86_64-linux-gnu/rustc1.16.0 (I've noticed this behaviour on 1.5.5 and upgraded to latest(1.6.6))

- Network: Kovan

- Config: parity --chain=kovan --rpcaddr=10.0.1.136 --rpcapi=web3,eth,personal --no-download --force-ui --ui-no-validation --ui-interface 0.0.0.0 --dapps-interface 10.0.1.136

**Comment by tomusdrw** [comment]
Could you run the node with `-lown_tx=trace,rpc=trace` to see if the transaction is indeed received? Do you Does the account you send ETH from is always unlocked or do you use `personal_sendTransaction`?

**Comment by bgadmin** [comment]
I've started node with "-lown_tx=trace,rpc=trace" will have to wait for unexpected behaviour to happen and I will post back results. 

I always unlock the account and use personal_sendTransaction.

Btw we had problems with keeping accounts unlocked for a longer period of time, don't know if this is a security issue, just letting you know.

**Comment by bgadmin** [comment]
@tomusdrw Could this maybe occur if I used "stale" version of parity?

**Comment by tomusdrw** [comment]
With `personal_sendTransaction` you don't need to unlock the account, you just provide a password with the transaction details (it's actually the safest method): https://github.com/paritytech/parity/wiki/JSONRPC-personal-module#personal_sendtransaction

Time-unlocking is supported only in `--geth` compatibility mode, it's very unsafe and should be avoided if possible. Parity will unlock the account only for a single transaction.

Regarding the original issue: Logs should reveal a lot of details about what exactly is happening on your node (please remove your passwords from the before submitting).

**Comment by bgadmin** [comment]
How could this work with web3 js library?

**Comment by tomusdrw** [comment]
The method should be available in web3.js, if it's not you can easily extend it using:
```js
web3._extend({
  property: 'personal',
  methods: [new web3._extend.Method({
    name: 'sendTransaction',
    call: 'personal_sendTransaction',
    params: 2,
    inputFormatter: [web3._extend.formatters.inputTransactionFormatter, (x) => String(x)]
  })]
});
```
